javascript/javascript2/week1/averageMedian/average_median.js

```diff
@@ -0,0 +1,68 @@
+const housePrices = [3000000,  3500000,  1300000,  40000000,  100000000,  8000000, 2100000];
+
+// to calculate average
+function getAverage(housePrices){
+
+  let sumHousePrice=0;
+
+  for(let i=0; i<housePrices.length; i++){
+
+      sumHousePrice+=housePrices[i];
+  }
```

Comment:
Can be written in two lines by using "Reduce" function. Use Reduce function to calculate the sum.